docker/passwd

```diff
@@ -0,0 +1 @@
+root:x:0:0:root:/root:/bin/bash
```

Comment:
nice

Reply:
Interesting pattern to provide our own `passwd` file, is there a reason for using this over something like `docker exec -u 0` which runs with you as root in the container? We could also `chpasswd` inside of the container as part of the Dockerfile's build perhaps? Never seen anyone provide their own `passwd` file but I'm not completely opposed to it, just curious about it.

Reply:
@wraithan Alok asked about this too. Actually it was just to avoid modifying the passwd file in-place to set root's shell to /bin/bash. Then it also became about hardening the container by removing unneeded accounts.

But now that you mention it, that seems too heavy-handed an approach. I changed it to calling `RUN usermod --shell /bin/bash root` instead.